ui/test/nuxeo-layout.test.js

```diff
@@ -27,57 +35,109 @@ const base = url.substring(0, url.lastIndexOf('/'));
 // Export Polymer and PolymerElement for 1.x and 2.x compat
 window.Polymer = Polymer;
 
-suite.skip('<nuxeo-layout>', () => {
-  test('layout not found', async () => {
-    const layout = await fixture(
-      html`
-        <nuxeo-layout href="notfound.html"></nuxeo-layout>
-      `,
-    );
-    await waitForEvent(layout, 'iron-resize');
-    expect(layout.$$('nuxeo-error').hidden).to.be.false;
-    expect(layout.element).to.be.undefined;
-  });
+suite('<nuxeo-layout>', () => {
+  suite('Error handling', () => {
+    test('Should display nuxeo-error when layout is not found', async () => {
+      const layout = await fixture(
+        html`
+          <nuxeo-layout href="notfound.html"></nuxeo-layout>
+        `,
+      );
+      await flush();
 
-  test('layout stamped with model', async () => {
-    const layout = await fixture(html`
-      <nuxeo-layout href="${base}/dummy-layout.html" model='{"text": "dummy"}'></nuxeo-layout>
-    `);
-    await waitChanged(layout, 'element');
-    expect(layout.$$('nuxeo-error').hidden).to.be.true;
-    expect(layout.element.localName).to.equal('dummy-layout');
-    expect(layout.element.text).to.equal('dummy');
-  });
+      const container = layout.$$('#container');
+      const nuxeoError = layout.$$('nuxeo-error');
+      if (!isElementVisible(nuxeoError)) {
+        await waitForAttrMutation(nuxeoError, 'hidden', null);
+      }
+      expect(isElementVisible(container)).to.be.false;
+      expect(isElementVisible(nuxeoError)).to.be.true;
+    });
 
-  test('layout valid field invalid', async () => {
-    const layout = await fixture(html`
-      <nuxeo-layout href="${base}/dummy-layout.html" model='{"text": "valid"}'></nuxeo-layout>
-    `);
-    const validity = await layout.validate();
-    expect(validity).to.equal(false);
-  });
+    test('Should display error messages', async () => {
+      const layout = await fixture(
+        html`
+          <nuxeo-layout href="notfound.html"></nuxeo-layout>
+        `,
+      );
+      await flush();
 
-  test('layout valid field valid', async () => {
-    const layout = await fixture(html`
-      <nuxeo-layout href="${base}/dummy-layout.html" model='{"text": "valid", "required": "foo"}'></nuxeo-layout>
-    `);
-    const validity = await layout.validate();
-    expect(validity).to.equal(true);
+      const nuxeoError = layout.$$('nuxeo-error');
+      const errorMessage = nuxeoError.$$('div.message');
+      if (!errorMessage) {
+        await waitForAttrMutation(errorMessage, 'hidden', false);
+      }
+      expect(nuxeoError.code).to.be.equal('404');
+      expect(nuxeoError.message).to.be.equal('Failed to find layout');
+      expect(nuxeoError.url).to.be.equal(layout.href);
```

Comment:
Do we want to test the bindings? Probably it would be more interesting to access the content, for consistency purposes.